ui/tests/acceptance/evaluations-test.js

```diff
@@ -328,6 +307,81 @@ module('Acceptance | evaluations list', function (hooks) {
         .dom('[data-test-empty-evaluations-list]')
         .exists('Renders a message saying no evaluations match filter status');
     });
+
+    test('it should enable combining filters and search', async function (assert) {
+      assert.expect(5);
```

Comment:
Out of curiosity, what is this assertion testing?

Reply:
Its all about testing the request params since everything combines via the `filter` query parameter now instead of individual qps.

Given a user combines multiples filters and search, when they add a new filter (or remove a filter) then we send the appropriate request to the server.

Reply:
Oh sorry, I meant specifically this line `assert.expect(5);`

Reply:
It counts the number of assertions in the test when we have an assertion that is in an asynchronous callback.